tokio/src/util/bit.rs

```diff
@@ -53,6 +43,10 @@ impl Pack {
         (base & !self.mask) | (value << self.shift)
     }
 
+    pub(crate) fn pack_lossy(&self, value: usize, base: usize) -> usize {
```

Comment:
nit: may be worth a comment explaining what this does, particularly that bits may be lost on both the least- and most-significant ends of `value`?